renku_notebooks/util/kubernetes_.py

```diff
@@ -183,4 +183,4 @@ def get_server_url(pod):
 
 
 def read_namespaced_pod_log(pod_name):
-    return v1.read_namespaced_pod_log(pod_name, kubernetes_namespace)
+    return v1.read_namespaced_pod_log(pod_name, kubernetes_namespace, tail_lines=100)
```

Comment:
What is this 100? Is this the maximum number of lines that you need in UI? Would you please rename it to something more expressive and use the named-constant here (e.g. `MAX_REQUIRED_LINES = 100`)?

Reply:
Yes, `tail_lines` is the maximum number of lines (from the most recent) that the function will return. We don't want to flood the UI with useless logs.

If you prefer a more explicit naming we may introduce it as an optional parameter like this

```
def read_namespaced_pod_log(pod_name, max_log_lines = 100):
    return v1.read_namespaced_pod_log(pod_name, kubernetes_namespace, tail_lines = max_log_lines)
```

Reply:
Both options are fine with me since we don't pass this value from REST API to this function yet.

Reply:
I am wondering if it makes sense to add the value directly to the rest API. We won't use this now, but at least we provide the possibility

Reply:
I know. That's why I'm saying we can have this either as a named-constant or a parameter. Since we don't pass the parameter, having it as a named-constant is OK as well.